### DISCUSSION TITLE
ONNX+Olive Installation Guide

### DISCUSSION THREAD

**Discussion by waylaa** [vladmandic/sdnext#2739] [Q&A] [answered]
Is there a step by step guide on how to install SD.Next with ONNX and Olive?

**Comment by vladmandic** [comment, marked as answer]
https://github.com/vladmandic/automatic/wiki/ONNX-Olive

**Reply by FrakerKill** [reply]
Nothing about Windows or Unix installation, cause I install Olive in Windows into Automatic and I can't detect AMD GPU and we can't use directml
